test/container/1.6.0/Dockerfile.gpu

```diff
@@ -0,0 +1,209 @@
+# Note: We need to install NCCL, cuDNN, and CUDA libraries since we are using base container.
```

Comment:
the intent with the test Dockerfiles is to make them as bare-bones as possible. I think it would make more sense here to either add to https://github.com/aws/sagemaker-mxnet-training-toolkit/blob/master/test/container/1.6.0/Dockerfile.dlc.gpu or create a new one that's similar to https://github.com/aws/sagemaker-mxnet-training-toolkit/blob/master/test/container/1.6.0/Dockerfile.mxnet.cpu

Reply:
Alright. assuming the 16.0-gpu-p3 docker image is made out of the dockerfile present in docker/1.6.0/Dockerfile.gpu this needs a couple of lines to be added for OpenSSH communication for MPI. Let me add it to the DLC.gpu and refer that.

Reply:
763104351884 is this account for DLC containers? since I was told containers are moved to DLC

Reason why I'm asking this is
There are patches made to DLC Dockerfile.gpu [2] which aren't in the SM-MX Dockerfile.gpu [1]


1. https://github.com/aws/sagemaker-mxnet-training-toolkit/blob/master/docker/1.6.0/py3/Dockerfile.gpu

2. https://github.com/aws/deep-learning-containers/blob/master/mxnet/training/docker/1.6.0/py3/Dockerfile.gpu

Reply:
> 763104351884 is this account for DLC containers? since I was told containers are moved to DLC

yes

> Reason why I'm asking this is
There are patches made to DLC Dockerfile.gpu [2] which aren't in the SM-MX Dockerfile.gpu [1]

[2] is the source of truth now, and [1] will be removed as we finish revamping our repos.

Reply:
Facing errors while reusing the dockerfiles.